SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -667,7 +667,7 @@ ERPCResult USpatialSender::SendRPCInternal(UObject* TargetObject, UFunction* Fun
 								*TargetActor->GetName(),
 								*ConnectionOwner->GetName()
 							)
-							bCanPackRPC = false;
+								bCanPackRPC = false;
```

Comment:
Spacing seems off - did you run a linter over this?

Reply:
This is bizzare, basically I ending up reverting files and did so by copying the text from the GitHub pages into my IDE which seems to have result in these, will fix.

Reply:
Turns out this happened because the UE_LOG on previous line doesn't have a semi-colon after, heh